source/common/http/conn_pool_grid.cc

```diff
@@ -324,8 +321,7 @@ ConnectionPool::Cancellable* ConnectivityGrid::newStream(Http::ResponseDecoder&
     }
   } else {
     // Make sure the HTTP/2 pool is created.
```

Comment:
I think we can probably delete this comment now since:

`pool = getOrCreateHttp2Pool()` 

is super readable. Woo hoo!

Reply:
done